README.md

```diff
@@ -3,14 +3,100 @@
 
 This is the Engine.IO Server Library for Java ported from the [JavaScript server](https://github.com/socketio/engine.io).
 
+See also: [Engine.IO-client Java](https://github.com/socketio/engine.io-client-java)
+
+## Installation
+This section will be updated when artifact is available on Maven Central.
+
+### Maven
+This section will be updated when artifact is available on Maven Central.
+
+### Gradle
+This section will be updated when artifact is available on Maven Central.
+
 #### Engine.IO Protocol 1.x suppport
 
 The current version of engine.io-java does not support protocol 1.x.
 
 ## Usage
 Usage is slightly different based on the server being used.
 
-TODO: Add usage for Jetty and Tomcat.
+Create a servlet to handle the HTTP requests as follows:
+```java
+public class EngineIoServlet extends HttpServlet {
+
+    private static final EngineIoServer ENGINE_IO_SERVER = new EngineIoServer();
```

Comment:
Is `static` mandatory here? Is it possible to create multiple servlet with the same underlying `EngineIoServer`?

Reply:
Creating the servlet is left up to the user. Normally, only one instance of EngineIoServer should exist per servlet instance. Multiple instances are possible if the user so desires. In this example, I've set it to static to be able to access the EngineIoServer instance from outside the servlet.

Reply:
Let's not encourage the reader to create a global variable then :+1: